cmd/cloud/cluster.go

```diff
@@ -18,7 +18,6 @@ func init() {
 	clusterCreateCmd.Flags().String("size", "SizeAlef500", "The size constant describing the cluster.")
 	clusterCreateCmd.Flags().String("zones", "us-east-1a", "The zones where the cluster will be deployed. Use commas to separate multiple zones.")
 	clusterCreateCmd.Flags().Int("wait", 600, "The amount of seconds to wait for k8s to become fully ready before exiting. Set to 0 to exit immediately.")
-	clusterCreateCmd.MarkFlagRequired("size")
```

Comment:
@gabrieljackson, I know we talked about this -- let me know if you disagree with this thinking, but I opted to make `--size` not required since we already default other bits in a non-trivial way, and it really simplifies the CLI to get a cluster going from scratch.